cosmos/models/user/__init__.py

```diff
@@ -0,0 +1,59 @@
+from django.contrib.auth.models import User
+from django.db import models
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from cosmos.models.user.constants import DEPARTMENTS, NATIONALITIES, PROGRAMS
+
+
+class Profile(models.Model):
```

Comment:
Why is profile not in it's own file?

Reply:
because the module only has one class

Reply:
I dont think it is good practice to put this in __init__.py. While possible, it causes confusion and is not really what the init file is meant for.

Reply:
in that case, I'll move it to a separate file, and just define `__all__` in `__init__`